sources/academy/platform/get_most_of_actors/naming_your_actor.md

```diff
@@ -21,23 +21,23 @@ For Actors such as [YouTube Scraper](https://apify.com/bernardo/youtube-scraper)
 
 GOOD:
 
-- Technical name (Actor's name in the [Apify Console](https://console.apify.com)): **${domain}-scraper**, e.g. **youtube-scraper**.
-- Publication title for the Apify Store: **${Domain} Scraper**, e.g. **YouTube Scraper**.
-- Name of the GitHub repository: **actor-${domain}-scraper**, e.g. **actor-youtube-scraper**.
+- Technical name (Actor's name in the [Apify Console](https://console.apify.com)): **$\{domain\}-scraper**, e.g. **youtube-scraper**.
+- Publication title for the Apify Store: **$\{Domain\} Scraper**, e.g. **YouTube Scraper**.
```

Comment:
iirc we only need to escape the opening bracket, the upgrading guide kinda confirms this

https://docusaurus.io/docs/migration/v3#bad-usage-of-